waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -1507,11 +1519,14 @@
                                       (do
                                         (log/warn "scheduler instance has not yet been initialized")
                                         {})))
+        syncer-timer-chan (scheduler/scheduler-syncer-timer-chan scheduler-syncer-interval-secs)
+        watch-chan-throttle-interval-ms (or watch-chan-throttle-interval-ms 1000)
```

Comment:
I think this should be tracked in the config.edn

Reply:
It already is, the `1000` is the default value when it is not explicitly configured.

There are other similar fields that have defaults provided similarly in this snippet, e.g. `pdb-api-version`, `restart-kill-threshold`.

Reply:
Why have a default value here instead of asserting those values are not nil and are specified in configuration?

Reply:
For backward compatibility :)